packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.jsx

```diff
@@ -79,10 +79,8 @@ const ContractInfo = ({
         if (is_turbos) {
             return (
                 <Localize
-                    i18n_default_text='<0>{{title}}</0> {{message}}'
-                    components={[<Text key={0} weight='bold' size='xxs' />]}
+                    i18n_default_text='{{message}}'
```

Comment:
@henry-deriv If message comes from BE it will be already translated, so maybe there is no need in using `<Localize />` here?